src/theory/bv/bv_solver_bitblast.h

```diff
@@ -89,6 +90,15 @@ class BVSolverBitblast : public BVSolver
    */
   void handleEagerAtom(TNode fact, bool assertFact);
 
+  /** Converts DRAT proof in terms of SAT literals to the corresponding Node
+   * literals.
+   *
+   * The result is a Node vector where each Node is an s-expression with a
+   * marker of whether it's a clause being added or deleted followed by the
+   * Node-level literals.
+   */
+  std::vector<Node> getProofNodes(proof::DratProof dratProof);
```

Comment:
```suggestion
  std::vector<Node> convertDratProof(proof::DratProof dratProof);
```
Also rename in the cpp.